sources/osgAnimation/BasicAnimationManager.js

```diff
@@ -429,9 +457,11 @@ define( [
                 instanceChannel.t = t; // reset time
                 instanceChannel.instanceAnimation = instanceAnimation; // link with parent animation
                 var targetID = instanceChannel.targetID;
+
+                if ( targetID === -1 ) continue;
```

Comment:
use a "js define" to encapsulate the magic number ? 
Animation.TARGET_UNDEFINED or something.
(allow for better search, does document what is the value, etc)